helper.linux.fish

```diff
@@ -1234,7 +1241,7 @@ end
 function buildUbuntuBuildImage3
   pushd $WORKDIR
   and cd $WORKDIR/containers/buildUbuntu3.docker
-  and docker build --pull -t $UBUNTUBUILDIMAGE3 .
+  and eval "docker build $IMAGE_ARGS --pull -t $UBUNTUBUILDIMAGE3 ."
```

Comment:
@KVS85 why is eval need here?

Reply:
Explained in https://github.com/arangodb/oskar/pull/356/commits/c4a64e834c59464d5e1bdfce288bde0d4bab4ce5.